core/modules/parsers/wikiparser/rules/syslink.js

```diff
@@ -0,0 +1,45 @@
+/*\
+title: $:/core/modules/parsers/wikiparser/rules/syslink.js
+type: application/javascript
+module-type: wikirule
+
+Wiki text inline rule for system tiddler links.
+Can be suppressed preceding them with `~`.
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "syslink";
+exports.types = {inline: true};
+
+exports.init = function(parser) {
+	this.parser = parser;
+	// Regexp to match
+	this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg;
```

Comment:
> Me neither. If you find this interesting to explore, please do.

It's a blocker for accepting this pull request. It means that the pull request has a failure mode that we don't fully understand. That makes it risky to accept.